test/test_md2html.py

```
@@ -2,22 +2,29 @@
 test_cases = [
     ('example.md', 'example.html'),
     ('edge_cases.md', 'edge_cases.html'),
+    ('inline_code_html.md', 'inline_code_html.html'),
 ]
 
 def run_converter(input_md, output_html):
-    with open(input_md, 'r', encoding='utf-8') as infile, open(output_html, 'w', encoding='utf-8') as outfile:
-        subprocess.run(['python', script, input_md], stdout=outfile, check=True)
+    input_path = os.path.join(BASE_DIR, input_md)
+    output_path = os.path.join(BASE_DIR, output_html)
+    with open(input_path, 'r', encoding='utf-8') as infile, open(output_path, 'w', encoding='utf-8') as outfile:
```

Ruff (0.11.9)

Local variable `infile` is assigned to but never used

Remove assignment to unused variable `infile`

(F841)